packages/perspective-viewer-highcharts/test/results/results.json

```diff
@@ -63,7 +63,7 @@
     "treemap.html/sorts by a hidden column.": "361cb1c47b5a6710319904c9c270101f",
     "treemap.html/sorts by a numeric column.": "4ac1683b5e1d68a2423e15e2b846b8c3",
     "treemap.html/filters by a numeric column.": "514acc4c586b399ba8e3bb50499c51c0",
-    "treemap.html/filters by a datetime column.": "ed5a3fde30646d63473d9c61cb7fe396",
```

Comment:
I guess we should drop the highcharts one from the PR?

Reply:
I have restored the original file.